actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -784,7 +784,7 @@ def link_to_remote_options?(options)
         end
 
         def add_method_to_attributes!(html_options, method)
-          if method_not_get_method?(method) && !html_options["rel"]&.match?(/nofollow/)
+          if method_not_get_method?(method) && !html_options["rel"]&.include?("nofollow")
```

Comment:
This also seems like an odd choice. Maybe fine but it's in the wrong PR.

Reply:
(linking for context) `Performance/StringInclude` was enabled in #45865

Reply:
Gotcha. Not sure about this one either! But yes, should be in compliance with the rules as they currently are. Just think that needs to happen in a clear PR separate from a dependency bump.